src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -797,9 +797,6 @@ function getTransformHandles(transformBounds, margin) {
 }
 
 function getTransformSelectionBounds(glyph, selection) {
-  if (selection.size <= 1) {
```

Comment:
The reason this is there is to avoid showing resize handles for a single-point selection. You need to distinguish between component and point selection.

Reply:
A single point will be covered anyway a view lines below.

Reply:
```
  if (width == 0 && height == 0) {
    // return undefined if for example only one point is selected
```